proto/ibc/applications/interchain_accounts/host/v1/tx.proto

```diff
@@ -33,3 +36,22 @@ message MsgUpdateParams {
 
 // MsgUpdateParamsResponse defines the response for Msg/UpdateParams
 message MsgUpdateParamsResponse {}
+
+// MsgModuleQuerySafe defines the payload for Msg/ModuleQuerySafe
+message MsgModuleQuerySafe {
+  option (cosmos.msg.v1.signer) = "signer";
+
+  option (gogoproto.goproto_getters) = false;
+
+  // signer address
+  string signer = 1;
+
+  // requests defines the module safe queries to execute.
+  repeated QueryRequest requests = 2;
+}
+
+// MsgModuleQuerySafeResponse defines the response for Msg/ModuleQuerySafe
+message MsgModuleQuerySafeResponse {
+  // protobuf encoded responses for each query
+  repeated bytes responses = 1;
+}
```

Comment:
```suggestion
  // block height of the query
  uint64 height = 2;
}
```
I think it might be helpful to add the block height at which the queries were executed.

Reply:
added